Yoast/wordpress-seo
# Add output escaping to collapsible content

## Summary

This PR can be summarized in the following changelog entry:

* N/A: improves output escaping.

## Relevant technical choices:

* `$collapsible->content()` contains html. An example is: 
```
<div class="paper tab-block metabox wpseo-form wpseo-collapsible-container" id="wpseo-collapsible-facebook">

	<h2 class="collapsible-header"><button id="wpseo-collapsible-facebook-button" type="button" class="toggleable-container-trigger" aria-expanded="false">Facebook <span class="toggleable-container-icon dashicons dashicons-arrow-down-alt2" aria-hidden="true"></span></button></h2>	<div id="wpseo-collapsible-facebook-container" class="paper-container toggleable-container toggleable-container-hidden"><div class="notice inline yoast-notice yoast-notice-go-premium">
				<p>Do you want to preview what it will look like if people share this post on Facebook? You can, with <strong>Yoast SEO Premium</strong>.</p>
				<p><a href="https://yoa.st/179?php_version=7.0&#038;platform=wordpress&#038;platform_version=5.1.1&#038;software=free&#038;software_version=11.8-RC1&#038;days_active=6-30&#038;user_language=en_US" target="_blank">Find out why you should upgrade to Yoast SEO Premium</a></p>
			</div><label for="yoast_wpseo_opengraph-title">Facebook Title</label><input type="text" id="yoast_wpseo_opengraph-title" name="yoast_wpseo_opengraph-title" value="" class="large-text" aria-describedby="yoast_wpseo_opengraph-title-desc"/><p id="yoast_wpseo_opengraph-title-desc" class="yoast-metabox__description">If you don't want to use the post title for sharing the post on Facebook but instead want another title there, write it here.</p><label for="yoast_wpseo_opengraph-description">Facebook Description</label><textarea class="large-text" rows="3" id="yoast_wpseo_opengraph-description" name="yoast_wpseo_opengraph-description" aria-describedby="yoast_wpseo_opengraph-description-desc"></textarea><p id="yoast_wpseo_opengraph-description-desc" class="yoast-metabox__description">If you don't want to use the meta description for sharing the post on Facebook but want another description there, write it here.</p><label for="yoast_wpseo_opengraph-image">Facebook Image</label><input id="yoast_wpseo_opengraph-image" type="text" size="36" class="" name="yoast_wpseo_opengraph-image" value="" aria-describedby="yoast_wpseo_opengraph-image-desc" readonly="readonly" /><input id="yoast_wpseo_opengraph-image_button" class="wpseo_image_upload_button button" data-target="yoast_wpseo_opengraph-image" data-target-id="yoast_wpseo_opengraph-image-id" type="button" value="Upload Image" /> <input class="wpseo_image_remove_button button" type="button" value="Clear Image" /><p id="yoast_wpseo_opengraph-image-desc" class="yoast-metabox__description">If you want to override the image used on Facebook for this post, upload / choose an image here. The recommended image size for Facebook is 1200 by 630 pixels.</p><input type="hidden" id="yoast_wpseo_opengraph-image-id" name="yoast_wpseo_opengraph-image-id" value=""/>
</div>
</div>
```
* `wp_kses_post` doesn't allow `input`, `select`, and `option` tags. That is why I introduced `extend_kses_post_with_forms` to allow those tags as well.
* We were also missing some a11y-related attributes, which is why I also introduced `extend_kses_post_with_a11y` to add those.
* To keep all HTML we need intact, I use `wp_kses_post` after I've added the abovementioned tags/attributes to the allowed tags/attributes list.

## Test instructions

This PR can be tested by following these steps:

* Open a post.
* Visit the social tab.
* See that the Twitter and Facebook collapsibles are still there, complete and functioning.
* See that all metabox tabs still have titles and icons. 
* See that the advanced collapsible still has its content.
* Merge this branch into premium.
* See that the Twitter and Facebook collapsibles are still there, complete and functioning.

## UI changes

## Documentation

## Quality assurance


Fixes #
